extensions/data-plane/data-plane-azure-storage/src/test/java/org/eclipse/edc/connector/dataplane/azure/storage/pipeline/AzureStorageDataSinkTest.java

```diff
@@ -68,6 +66,8 @@ class AzureStorageDataSinkTest {
     private final String content = "Test Content";
     private final Exception exception = new TestCustomException("Test custom exception message");
     private final BlobMetadataProvider metadataProvider = new BlobMetadataProviderImpl(monitor);
+    private final DestinationBlobName destinationBlobName = new DestinationBlobName();
```

Comment:
please use a mock and test collaboration, otherwise this test is an integration one, and we want to avoid them unless strictly necessary.